cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRunnerClassLoader.java

```diff
@@ -181,4 +191,106 @@ protected void onMethodExit(int opcode) {
     byte[] byteCode = cw.toByteArray();
     return defineClass(name, byteCode, 0, byteCode.length);
   }
+
+  private Class<?> defineAkkaRemoting(String name,
```

Comment:
Can you add a comment here as to what code it is looking for and what it replaces it with, and why?